selfdrive/manager.py

```diff
@@ -526,9 +526,6 @@ def uninstall():
 def main():
   os.environ['PARAMS_PATH'] = PARAMS
 
-  # the flippening!
-  os.system('LD_LIBRARY_PATH="" content insert --uri content://settings/system --bind name:s:user_rotation --bind value:i:1')
```

Comment:
Seems like this is done in NEOS now?

Reply:
I'd have to check that. Can you leave it in for now?

Reply:
done